xla/service/gpu/kernels/BUILD

```diff
@@ -120,7 +121,7 @@ xla_test(
 
 cc_library(
     name = "topk_custom_kernel",
-    srcs = if_cuda_is_configured(
+    srcs = if_cuda_or_rocm(
```

Comment:
Please use if_gpu_is_configured, otherwise it will require --config=cuda or --config=rocm to run. We try to avoid that inside XLA.

Reply:
Good point. Yes, that would be great if you can add that, we will need it here.

Reply:
I happens to be messy to provide if_gpu_is_configured with else branch, so I unified everything into a single file.